nixos/modules/services/web-apps/glance.nix

```
@@ -99,6 +99,16 @@ in
       '';
     };
 
+    configFile = mkOption {
+      type = types.nullOr types.path;
+      default = null;
+      example = lib.literalExpression "./glance.yaml";
+      description = ''
+        Path to a YAML configuration file for Glance.
+        If set, `services.glance.settings` will be ignored.
+      '';
```

Mentioning upstream docs here too would be nice:
```suggestion
      description = ''
        Path to a YAML configuration file for Glance.
        See <https://github.com/glanceapp/glance/blob/main/docs/configuration.md> for more.

        If set, `services.glance.settings` will be ignored.
      '';
```